digger/builds/cordova_android.py

```diff
@@ -56,3 +58,51 @@ def get_export_path(self):
     Gets the apk(s) path that can be exported outside of the container.
     """
     return ','.join(android_helper.find_apks(self.path))
+
+
+class CordovaLightAndroidBuild(CordovaAndroidBuild):
+  def __init__(self, tmp_folder=None, *args, **kwargs):
+    super(CordovaLightAndroidBuild, self).__init__(*args, **kwargs)
+    self.tmp_folder = tmp_folder
+    if self.tmp_folder is None:
+      self.tmp_folder = '%s_tmp' % self.path.split('/')[-1:][0]
```

Comment:
Maybe use some kind of path.join? This results in weird behavior when I build like this:
```
cd my-project
abcd build --path=.
> bla bla ...
> FileNotFoundError: [Errno 2] No such file or directory: '/._tmp/www/config.json'
```